styles.css

```diff
@@ -0,0 +1,370 @@
+
+    place-items: end;
+}
+
+.post-nav-button {
+    background-color: white;
+    border: none;
+    border-radius: 50px;
+    position: relative;
+    left: 50px;
+    bottom: 10px;
+    
+}
+
+.post-nav-button:hover {
+    background-color: rgba(49, 149, 236, 0.328);
+}
+
+.post-nav-button:active {
+    background-color: rgba(49, 149, 236, 0.796);
+}
+
+.post-nav {
+    width: 30px;
+    height: 30px;
+    padding-top: 3px;
+    padding-bottom: 3px;
+}
+
+.tweet-button2{
+    background-color: rgba(75, 160, 235, 0.373);
+    border: none;
+    border-radius: 50px;
+    margin: 10px;
+    height: 40px;
+    width: 80px;
+    color: white;
+    font-weight: bold;
+    grid-column: 7;
+}
+
+.tweet-button2:hover {
+    background-color: rgb(75, 160, 235);
+}
+
+.tweet-button2:active {
+    background-color: rgb(77, 172, 254) ;
+}
+.spacer {
+    background-color: rgba(128, 128, 128, 0.025);
+    border-top: rgb(128, 128, 128, 0.068) solid 1.2px;
+    border-bottom: rgba(128, 128, 128, 0.068) solid 1.2px;
+}
+
+.feed {
+    margin-right: 10px;
+    overflow-y: scroll;
+}
+
+.feed::-webkit-scrollbar{
+    display: none;
+}
+
+.feed-post-grid {
+    display: grid;
+    grid-template-columns: 70px 1fr;
+    margin-top: 20px;
+    padding-left: 20px;
+    border-bottom: rgba(128, 128, 128, 0.068) solid 1.2px;
+}
+
+.name-profile-settings {
+    display: grid;
+    grid-template-columns: .3fr .5fr 2fr .1fr;
+    color: rgb(0, 0, 0);
+}
+
+.name-profile-settings p{
+color: rgba(128, 128, 128, 0.695);
+}
+
+.name-profile-settings img {
+    grid-column: 4;
+    padding-right: 10px;
+}
+
+.content p {
+    padding-top: 5px;
+    padding-bottom: 5px;
+    font-family: sans-serif;
+    font-size: 15px;
+    color: black;
+}
+
+.content img{
+    width: 400px;
+    object-fit: contain;
+}
+
+.content-container{
+   display: grid;
+   grid-template-rows: auto;
+}
+
+.feed-nav {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    padding-top: 5px;
+}
+
+.feed-nav svg {
+    width: 18.75px;
+    height: 18.75;
+}
+
+.feed-nav button {
+    background: none;
+    border: none;
+    border-radius: 50px;
+    width: 90px;
+    color: rgba(128, 128, 128, 0.501);
+}
+
+.feed-nav button:hover {
+    background: rgba(75, 160, 235, 0.373);
+}
+
+.feed-nav p{
+    display: inline;
+    padding: 10px;
+    position: relative;
+    bottom: 5px;
+}
+
+/* RIGHT MENU */
+
+
+.right-menu {
+    display: grid;
+    grid-template-rows: 50px 1fr;
+    margin-left: 30px;
+}
+
+.search-bar {
+    display: grid;
+    grid-template-columns: 50px 1fr;
+    background-color: rgba(128, 128, 128, 0.191);
+    border-radius: 50px;
+    margin-top: 5px;
+    align-content: center;
+}
+
+.search-bar svg {
+    width: 25px;
+    padding-left: 7px;
+}
+
+.search-bar p {
+    color: grey;
+    font-family: sans-serif;
+    padding-top: 3px;
+}
+
+.whats-happening {
+    display: grid;
+    grid-template-rows:   50px repeat(auto-fit, 1fr) 50px; /* NEED TO FIX */
```

Comment:
Where you have wrote "1fr"; this could be changed to min-max(200px, 1fr) and I think it could help if you are having issues with spacing.

Reply:
Yah. Currently, the issue is getting exact sizes on the top and bottom. Working through it with Lewis.